igarss_2023/igarss_mtedgecumbe_ts_analysis.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p>Since we'll be in OpenSARLab we'll probably want to empasise this can be done anywhere, and maybe the benefits of being <em>in</em> OSL... it might also be worth looking at their conda helpers and docs around conda...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFHyP3/tutorials/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>